docs/sphinx/reference-frontend-api.rst

```diff
@@ -595,3 +599,27 @@ Functions
 .. function:: void obs_frontend_reset_video(void)
 
    Reloads the UI canvas and resets libobs video with latest data from profile.
+
+---------------------------------------
+
+.. function:: void obs_frontend_set_tbar_position(int position)
+
+   Sets the value of the transition bar.
+
+   :param position: The position to set the t-bar to
+   :value Range: * 0 - 1023
+
+---------------------------------------
+
+.. function:: void obs_frontend_release_tbar(void);
+
+   Release the transition bar.   (like a user releasing their mouse button after moving it).
```

Comment:
I'm not a fan of long parenthetical statements, especially with a period after.  Is the example here necessary?  If so, please rephrase it into a regular sentence.

Reply:
Any suggestions on this one?

Reply:
maybe 

`Release the T-bar - emulates mouse button release`

Reply:
I don't use the T-bar, so I had to quickly play with it to figure out what `obs_frontend_release_tbar` does.  It invokes [`OBSBasic::TBarReleased()`](https://github.com/obsproject/obs-studio/blob/e8a1c082daf7b79ec36aa6dfe3a079ba222f3e2b/UI/window-basic-main-transitions.cpp#L922), which appears to determine whether the current transition should be completed (it was dropped near the right), reset (it was dropped near the left), or held (it was dropped outside either of those thresholds), and then if needed, resets the T-bar to 0 and enables/unlocks the transition widgets.  So "releasing their mouse" is correctly describing the process, though it is simplified.

Something like this is a little more descriptive without getting into the weeds, hopefully:
> Emulate a mouse button release on the transition bar and determine transition status.